ova/provision.sh

```diff
@@ -1,14 +1,12 @@
 #!/bin/bash
 
-WAZUH_VERSION=$1
-OPENDISTRO_VERSION=$2
-ELK_VERSION=$3
-PACKAGES_REPOSITORY=$4
-BRANCH=$5
-BRANCHDOC=$6
-DEBUG=$7
-UI_REVISION=$8
-INSTALLER="all-in-one-installation.sh"
+
+PACKAGES_REPOSITORY=$1
+DEBUG=$2
+PACKAGES_BRANCH=$3
+
+INSTALLER="unattended-installation.sh"
+URL_INSTALLER="https://raw.githubusercontent.com/wazuh/wazuh-packages/${PACKAGES_BRANCH}/resources/open-distro/unattended-installation"
```

Comment:
Use the file inside the repository and branch.

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/832/commits/9d4132b545b62ad4399d421d3e37db7dc3a73ea2